README.md

```diff
@@ -62,46 +73,66 @@ entering the system) and matrices containing growth and survival rates for each
 `FunctionalGroup` and `SizeClass` is needed:
 
 ```julia
-using CoralBlox: timestep
-
-# Mock recruits
-recruits::Vector{Float64} = [1e5, 2e5]
+using CoralBlox: linear_extension_scale_factors, max_projected_cover,
+using CoralBlox: timestep!, coral_cover
 
 # Mock linear extensions
 linear_extensions::Matrix{Float64} = [
-    15 25 35 25;
-    10 20 25 15;
+    0.004 0.025 0.1 0.0;
+    0.003 0.007 0.005 0.0;
+    0.002 0.004 0.01 0.0
 ]
 
 # Mock survival rate
-survival_rate::Matrix{Float64} = 1 .- [
-    0.05 0.05 0.03 0.01;
-    0.1 0.08 0.04 0.02
+survival_rate::Matrix{Float64} = [
+    0.5 0.6 0.65 0.8;
+    0.6 0.7 0.8 0.8;
+    0.5 0.8 0.8 0.8
 ]
 
-for tstep::Int64 in 2:n_timesteps
-    # Linear extension scale factor
-    lin_ext_scale_factors::Vector{Float64} = linear_extension_scale_factors(
-        C_cover[tstep, :, :],
-        available_area,
-        linear_extensions,
-        size_class_bounds,
-        habitable_max_projected_cover,
-    )
+# Caclulate maximum projected cover
+habitable_max_projected_cover = max_projected_cover(
+    linear_extensions,
+    size_class_bounds,
+    habitable_area
+)
+
+# Only apply linear extension scale factor when cover is above the scale_threshold
+scale_threshold = 0.9 * habitable_area
+
+# Linear extension scale factor
+local lin_ext_scale_factors::Float64
```

Comment:
Consider using a more readable variable name. Then you can remove the comment 😄

Reply:
Fine if you want to leave it, just note that the variable is plural, but the value it holds is singular.